src/Entity/Field/CheckboxField.php

```diff
@@ -37,7 +38,7 @@ public function setValue($value): Field
         return parent::setValue($value);
     }
 
-    public function getTwigValue()
+    public function getTwigValue(): string|array|Markup|bool
```

Comment:
This can be scoped further. However, signature changes are a BC-break.

Reply:
Reverted.